mysql-test/suite/rpl/t/percona_bug_ps3885.test

```diff
@@ -0,0 +1,26 @@
+#
+# PS-3885: Assertion when switching slaves without stopping the slave thread
+#
+
+--let $rpl_topology= 1->2->3
+--source include/rpl_init.inc
+
+-- connection server_1
+SHOW MASTER STATUS;
+
+-- connection server_2
+SHOW MASTER STATUS;
+
+-- connection server_3
+STOP SLAVE IO_THREAD;
+--eval CHANGE MASTER TO MASTER_HOST='127.0.0.1', MASTER_PORT=$SERVER_MYPORT_1, MASTER_LOG_FILE='mysqld-1.000001';
```

Comment:
Potentially, `$SERVER_MYPORT_1` can be different between MTR runs, so it must be replaced with a placeholder
```
--replace_result $SERVER_MYPORT_1 SERVER_MYPORT_1
```